amethyst_rendy/src/bundle.rs

```diff
@@ -956,7 +982,14 @@ mod tests {
                         .with_color_surface()
                         .with_depth_stencil(depth),
                 )
-                .with_surface(surface2, None),
+                .with_surface(
+                    surface2,
+                    hal::window::Extent2D {
+                        width: 1,
+                        height: 1,
+                    },
```

Comment:
No idea how this should actually be set to in the test. This was necessary due to the API change in https://github.com/amethyst/rendy/pull/239.

Reply:
seems fine, if the test passes it's inconsequential